### PR TITLE
[Rule Tuning] Windows DR Tuning - 9

### PR DESCRIPTION
**Pull request by w0rk3r**
## Issues

Part of #3186 

## Summary

Tunes the following rules:

* Suspicious Execution via Windows Subsystem for Linux
* Execution via Windows Subsystem for Linux
* Enumeration of Administrator Accounts
* Account Discovery Command via SYSTEM Account
* PowerShell Suspicious Discovery Related Windows API Functions

### CODE REVIEW

File: rules/windows/defense_evasion_wsl_bash_exec.toml [hunk 1]
```diff
@@ -22,23 +22,31 @@ references = [
...
+      not process.command_line : ("bash", "bash.exe")
+    ) or 
+    process.executable : "?:\\Users\\*\\AppData\\Local\\Packages\\*\\rootfs\\usr\\bin\\bash" or 
+    (
+      process.parent.name : "wsl.exe" and process.parent.command_line : "bash*" and not process.name : "wslhost.exe"
+    ) or 
+    (
+      process.name : "wsl.exe" and process.args : (
+        "curl", "/etc/shadow", "/etc/passwd", "cat", "--system", "root", "-e", "--exec", "bash", "/mnt/c/*"
+      ) and not process.args : ("wsl-bootstrap", "docker-desktop-data", "*.vscode-server*")
```
Comment (w0rk3r) [inline rules/windows/defense_evasion_wsl_bash_exec.toml:46]: Actual changes to logic, the rest is to make it more readable

File: rules/windows/discovery_admin_recon.toml [hunk 2]
```diff
@@ -75,18 +75,20 @@ type = "eql"
...
-     process.args : ("group", "useraccount"))
-) and not user.id in ("S-1-5-18", "S-1-5-19", "S-1-5-20")
+  (
+    (
+      (process.name : "net.exe" or process.pe.original_file_name == "net.exe") or
+      ((process.name : "net1.exe" or process.pe.original_file_name == "net1.exe") and not process.parent.name : "net.exe")
+    ) and
+    process.args : ("group", "user", "localgroup") and
+    process.args : ("*admin*", "Domain Admins", "Remote Desktop Users", "Enterprise Admins", "Organization Management")
+    and not process.args : ("/add", "/delete")
```
Comment (w0rk3r) [inline rules/windows/discovery_admin_recon.toml:85]: Actual change to logic, the rest is to make it more readable

File: rules/windows/discovery_posh_suspicious_api_functions.toml [hunk 3]
```diff
@@ -111,7 +111,7 @@ event.category:process and host.os.type:windows and
     NetScheduleJobEnum or
     NetUserModalsGet
   )
-  and not user.id : ("S-1-5-18" or "S-1-5-19")
```
Comment (w0rk3r) [inline rules/windows/discovery_posh_suspicious_api_functions.toml:114]: As we have better visibility around PowerShell rules, I'm removing this broad exception, the rule will be tuned after the next release to fix new FPs

File: rules/windows/defense_evasion_wsl_bash_exec.toml (hunk 1 above)
Comment (Aegrah) [reply rules/windows/defense_evasion_wsl_bash_exec.toml:46]: Does it make sense to flag on `process.args` of `cat`, `root`, `-e`? To me that sounds a bit generic. I would suggest adding:

- [Tunneling related activity](https://github.com/elastic/detection-rules/blob/main/rules/linux/command_and_control_linux_tunneling_and_port_forwarding.toml)
- [Specific tools such as nc, or another utility that could potentially spawn reverse connections](https://github.com/elastic/detection-rules/blob/main/rules/linux/execution_shell_via_lolbin_interpreter_linux.toml)
- [File alterations in specific suspicious directories](https://github.com/elastic/detection-rules/blob/main/rules/linux/persistence_etc_file_creation.toml)
- Tools that can help initialize network connections such as `wget`. 

Might be a large change, LMK if you would like to add some of these changes. Can sync via Slack.

Comment (Aegrah) [reply rules/windows/defense_evasion_wsl_bash_exec.toml:46]: These additions will be added in seperate rules, and are tracked in https://github.com/elastic/detection-rules/issues/3362.

**Review by Aegrah — APPROVED** [review]
LGTM!

**Review by imays11 — APPROVED** [review]

**Pull request merged**